tommy-client/src/app/apiget.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 import { HttpClient, HttpHeaders } from '@angular/common/http';
-import { config } from './../environments/config.dev';
+import { config } from './../environments/config.dev-no-prox';
```

Comment:
Not all want to use the dev without proxy

Reply:
ok, we dont must to remove it